File: Selenium/Project/PActivity3.py
from selenium import webdriver
from selenium.webdriver.common.by import By
from selenium.webdriver.firefox.service import Service as FirefoxService
from webdriver_manager.firefox import GeckoDriverManager

# Set up the Firefox Driver with WebDriverManger
service = FirefoxService(GeckoDriverManager().install())

# Start the Driver
with webdriver.Firefox(service=service) as driver:
    # Open the browser to the URL
    driver.get("http://alchemy.hguy.co/crm")
    # Get the first copyright text in the footer
    CopyWriteText =driver.find_element(By.ID, "admin_options").text
    print("first copyright text in the footer: ", CopyWriteText)

File: Selenium/Project/PActivity3.java
package Project;

import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class PActivity3 {
    public static void main(String[] args) throws InterruptedException {
        // Setup the Firefox driver(GeckoDriver)
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
        WebDriverManager.firefoxdriver().setup();
        // Create a new instance of the Firefox driver
        WebDriver driver = new FirefoxDriver();
        driver.get("http://alchemy.hguy.co/crm");
        // Get the first copyright text in the footer
        String CopyWriteText = driver.findElement(By.id("admin_options")).getText();
        System.out.println("first copyright text in the footer: " + CopyWriteText);
        // Close the browser
        driver.quit();
    }
}
